auto_vk_toolkit/src/context_vulkan.cpp

```diff
@@ -911,6 +919,28 @@ namespace avk
 	}
 #endif
 
+	bool context_vulkan::supports_dynamic_rendering(const vk::PhysicalDevice& device)
+	{
+		vk::PhysicalDeviceProperties2 physicalProperties;
+		device.getProperties2(&physicalProperties, dispatch_loader_core());
+		bool dynamic_rendering_supported = false;
+
+		// Dynamic rendering is promoted to core in vulkan 1.3
+		if(VK_API_VERSION_MINOR(physicalProperties.properties.apiVersion) >= 3)
```

Comment:
I believe you are right. I put this as a temporary solution since I was unsure the else branch would work for 1.3. I did look into it and confirmed it still should work but looks like I still forgot the branch in, thanks for the catch!